rotkehlchen/chain/ethereum/transactions.py

```diff
@@ -42,6 +43,8 @@ def _graph_delegation_callback(
             self,
             last_block_queried: int,
             filters: dict[str, Any],
+            new_events: list[dict[str, Any]],  # pylint: disable=unused-argument
+            cb_arguments: 'LOG_ITERATION_CALLBACK_ARGUMENTS_TYPE',  # pylint: disable=unused-argument
```

Comment:
don't do it like this. Please have it explicit here. `'WithdrawalsQueryParameters | None`. Otherwise I have to jump around to understand this is optional. Using yet another "alias" makes it harder to read and understand